SimPEG/MT/Utils/MT1Danalytic.py

```diff
@@ -0,0 +1,111 @@
+# Analytic solution of EM fields due to a plane wave
+
+import numpy as np, SimPEG as simpeg
+
+def getEHfields(m1d,sigma,freq,zd,scaleUD=True):
+    '''Analytic solution for MT 1D layered earth. Returns E and H fields.
+
+    :param SimPEG.mesh, object m1d: Mesh object with the 1D spatial information.
+    :param numpy.array, vector sigma: Physical property of conductivity corresponding with the mesh.
+    :param float, freq: Frequency to calculate data at.
+    :param numpy array, vector zd: location to calculate EH fields at
+    :param bollean, scaleUD: scales the output to be 1 at the top, increases numeracal stability.
+
+    Assumes a halfspace with the same conductive as the last cell below.
+
+    '''
+    # Note add an error check for the mesh and sigma are the same size.
+
+    # Constants: Assume constant
+    mu = 4*np.pi*1e-7*np.ones((m1d.nC+1))
```

Comment:
you can use from `scipy.constants import mu_0` or once we merge dev onto master `from SimPEG.EM import mu_0`